.circleci/config.yml

```diff
@@ -40,3 +43,37 @@ jobs:
           | python3 - --version 1.1.0"
       - run: poetry build
       - run: POETRY_HTTP_BASIC_PYPI_PASSWORD=${PYPI_PASSWORD} poetry publish
+  test_installation:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install python3 python3-dev curl git -y
+      - checkout
+      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+          | python3 - --version 1.1.0"
+      - run: echo "export PATH=$PATH:$HOME/.poetry/bin" >> $BASH_ENV
+      - run: sudo $(which poetry) install
+      - run: sudo $(which poetry) run brainframe install --noninteractive
+      - run: sudo $(which poetry) run brainframe compose up -d
+      - run:
+          name: Check BrainFrame containers are running
+          command: |
+            for i in `seq 1 5`; do
```

Comment:
Minor thing: I believe backticks in Bash are discouraged in favor of using `$()`, so this would become `for i in $(seq 1 5); do`.